### ISSUE TITLE
Physical Description (`//physDesc`) display changes

### ISSUE THREAD

**Issue by wlpotter** [srophe/britishLibrary#37]
Note that some of the data is not present in the current app versions (waiting on #36 )

# Display

1. `//physDesc/p` as a paragraph but without the current "Physical Description" heading (this moves to the box label in another issue)
2. For `//physDesc/handDesc`, `//physDesc/decoDesc`, and `physDesc/additions`, we will create separate issues for these

# Hide

1. The `physDesc/objectDesc` can be hidden

**Comment by wlpotter** [comment]
Note to self: need to add more than this

**Comment by wsalesky** [comment]
Done

**Comment by davidamichelson** [comment]
physDesc/objectDesc appears to still be present?

**Comment by wsalesky** [comment]
@davidamichelson can you clarify? example record and what is showing that should be suppressed?

**Comment by wlpotter** [comment]
Looks like it's working now. Maybe the changes weren't showing up when we looked last week because of the webhooks issue?
